spec/system/jobseekers/jobseekers_can_create_a_job_alert_from_a_mailing_campaign_spec.rb

```diff
@@ -61,10 +61,6 @@
     expect(page).not_to have_checked_field("Teacher")
     expect(page).not_to have_checked_field("Full time")
 
-    validate_and_confirm
```

Comment:
I was going to change this method name to be something that showed what it actually did a little better but realised it is only used once so I jut removed it.

Reply:
:question: Wasn't this used twice?
The call is being deleted from 2 different scenarios (lines 35 and 64), but is now only being tested on the second scenario.

Reply:
All good then!